CHANGELOG

```diff
@@ -1,3 +1,8 @@
+== Version 4.4.0
```

Comment:
👍 to 4.4: https://github.com/Shopify/shopify_api/blob/master/RELEASING#L3
> Given a version number MAJOR.MINOR.PATCH, increment the:
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.